test/tools/ossfuzz/protoToYul.h

```diff
@@ -295,12 +309,16 @@ class ProtoConverter
 	}
 
 	std::ostringstream m_output;
-	/// Variables in current scope
-	std::stack<std::vector<std::string>> m_scopeVars;
+	/// Variables in all function definitions
+	std::vector<std::vector<std::vector<std::string>>> m_funcVars;
+	/// Variables in current function definition
+	std::vector<std::string const*> m_currentFuncVars;
```

Comment:
Does this need to be stored here, instead of just retrieved from the function when needed?

Reply:
It could be retrieved. I guess I used a class member to make it more convenient to access the data across class members without having to pass vector by reference.